lib/egree/version.rb

```diff
@@ -1,3 +1,3 @@
 module Egree
-  VERSION = "0.0.6"
```

Comment:
We should probably not bump version until we find it ready to relase. I find it good to make that as a commit just before it's tagged.